pkg/db/postgres.go

```diff
@@ -19,6 +18,29 @@ import (
 	"go.uber.org/zap"
 )
 
+type migrateLogger struct {
+	logger  *zap.SugaredLogger
+	verbose bool
+}
+
+func newMigrateLogger(debug bool) *migrateLogger {
+	logger := log.NewLogger(debug)
+	return &migrateLogger{
+		logger:  logger,
+		verbose: debug,
+	}
+}
+
+func (ml migrateLogger) Printf(format string, v ...interface{}) {
+	if ml.verbose {
+		ml.logger.Debugf(format, v...)
+	}
```

Comment:
As reminder: we have to also log something in non verbose mode

Reply:
Thanks! Applied 👍